<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.3 (Python 3.11.5 on win32)" generated="2025-06-10T02:53:02.288391" rpa="false" schemaversion="5">
<suite id="s1" name="Testes" source="C:\Users\vitor\Documents\GitHub\Seminario-C214\testes">
<suite id="s1-s1" name="Test Estoque" source="C:\Users\vitor\Documents\GitHub\Seminario-C214\testes\test_estoque.robot">
<test id="s1-s1-t1" name="Visualizar Estoque Com Produtos" line="5">
<kw name="Iniciar SessÃ£o" owner="keywords">
<kw name="Create Session" owner="RequestsLibrary">
<msg time="2025-06-10T02:53:02.851036" level="FAIL">Variable '${BASE_URL}' not found.</msg>
<arg>api</arg>
<arg>${BASE_URL}</arg>
<doc>Create Session: create a HTTP session to a server</doc>
<status status="FAIL" start="2025-06-10T02:53:02.850035" elapsed="0.001997">Variable '${BASE_URL}' not found.</status>
</kw>
<status status="FAIL" start="2025-06-10T02:53:02.848035" elapsed="0.003997">Variable '${BASE_URL}' not found.</status>
</kw>
<kw name="GET On Session" owner="RequestsLibrary">
<var>${resp}</var>
<arg>api</arg>
<arg>/estoque/</arg>
<doc>Sends a GET request on a previously created HTTP Session.</doc>
<status status="NOT RUN" start="2025-06-10T02:53:02.852032" elapsed="0.000000"/>
</kw>
<kw name="Should Be Equal As Integers" owner="BuiltIn">
<arg>${resp.status_code}</arg>
<arg>200</arg>
<doc>Fails if objects are unequal after converting them to integers.</doc>
<status status="NOT RUN" start="2025-06-10T02:53:02.853036" elapsed="0.000000"/>
</kw>
<kw name="Set Variable" owner="BuiltIn">
<var>${produtos}</var>
<arg>${resp.json()}</arg>
<doc>Returns the given values which can then be assigned to a variables.</doc>
<status status="NOT RUN" start="2025-06-10T02:53:02.853036" elapsed="0.000000"/>
</kw>
<kw name="Length Should Be Greater Than">
<arg>${produtos}</arg>
<arg>0</arg>
<status status="NOT RUN" start="2025-06-10T02:53:02.854036" elapsed="0.000000"/>
</kw>
<status status="FAIL" start="2025-06-10T02:53:02.846035" elapsed="0.009003">Variable '${BASE_URL}' not found.</status>
</test>
<status status="FAIL" start="2025-06-10T02:53:02.376164" elapsed="0.483872"/>
</suite>
<status status="FAIL" start="2025-06-10T02:53:02.292393" elapsed="0.570645"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Testes" id="s1" pass="0" fail="1" skip="0">Testes</stat>
<stat name="Test Estoque" id="s1-s1" pass="0" fail="1" skip="0">Testes.Test Estoque</stat>
</suite>
</statistics>
<errors>
<msg time="2025-06-10T02:53:02.838036" level="ERROR">Error in file 'C:\Users\vitor\Documents\GitHub\Seminario-C214\testes\resources\keywords.robot' on line 3: Processing variable file 'variables.robot' with arguments ['file'] failed: Importing variable file 'variables.robot' failed: ModuleNotFoundError: No module named 'variables'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\Scripts\robot.exe
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\python311.zip
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\DLLs
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\Lib
  C:\Users\vitor\AppData\Local\Programs\Python\Python311
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\Lib\site-packages
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\Lib\site-packages\win32
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\Lib\site-packages\win32\lib
  C:\Users\vitor\AppData\Local\Programs\Python\Python311\Lib\site-packages\Pythonwin</msg>
</errors>
</robot>
